README.md

```diff
@@ -61,6 +61,8 @@ For more configuration options, see [Configuration](#configuration)
 | `npm run dev`          | Starts backend server and client app in dev mode, which watches for changes in the source files and automatically rebuilds/restarts the corresponding components. |
 | `npm test`             | Run tests.                                                                                                                                                        |
 
+To add dependencies, you can use `scope=[scope-without-@padloc/] npm run add [package]` and to remove them, run `scope=[scope-without-@padloc/] npm run remove [package]`.
```

Comment:
So now we have these two commands. They're not as "sturdy" as if they were `Makefile` or `.sh`, but they work and make these actions simpler and easier to make.

For reference, here's what I had to do to remove `1pux-to-csv` and add `jszip` to `@padloc/app`:

```bash
scope=app npm run remove 1pux-to-csv
scope=app npm run add jszip
```

Reply:
I think I'd prefer
 ```sh
npm run add/remove <package> [scope]
```
What do you think?

Reply:
Yeah, I tried that initially, but `npm` messes up `$2`, so it needs to be set as a variable before the script runs, unfortunately.